src/protocols/BaseProtocolAdapter.ts

```diff
@@ -19,14 +20,18 @@ export abstract class BaseProtocolAdapter {
 
   abstract protocolName: string;
 
+  abstract coinSymbol: string;
+
+  abstract coinSymbolShort: string;
```

Comment:
```suggestion
  abstract coinName: string;

  abstract coinSymbol: string;
```

Maybe just coin name and coin symbol?